File: java/javadpbo/src/com/sem4dpbo/week2/contohkasus/Pengusaha.java
package com.sem4dpbo.week2.contohkasus;

public class Pengusaha {
    private int nomorUsaha;
    private String nama, alamat, namaUsaha;

    Pengusaha(){}

    Pengusaha(String nama, String alamat, String namaUsaha, int nomorUsaha){
        this.nomorUsaha = nomorUsaha;
        this.nama = nama;
        this.alamat = alamat;
        this.namaUsaha = namaUsaha;
    }

    //method set
    public void setNomorUsaha(int nomorUsaha) {
        this.nomorUsaha = nomorUsaha;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public void setAlamat(String alamat) {
        this.alamat = alamat;
    }

    public void setNamaUsaha(String namaUsaha) {
        this.namaUsaha = namaUsaha;
    }

    //method get
    public int getNomorUsaha() {
        return nomorUsaha;
    }

    public String getNama() {
        return nama;
    }

    public String getAlamat() {
        return alamat;
    }

    public String getNamaUsaha() {
        return namaUsaha;
    }
}

File: cpp/2nd_Week/ContohKasusKayu/Pengusaha.cpp
#include <iostream>
#include <string>

using namespace std;

class Pengusaha
{
    private:
        /* data */
        int nomorUsaha;
        string nama, alamat, namaUsaha;

    public:
        Pengusaha(){};

        Pengusaha(string nama, string alamat, string namaUsaha, int nomorUsaha){
            this->nomorUsaha = nomorUsaha;
            this->nama = nama;
            this->alamat = alamat;
            this->namaUsaha = namaUsaha;
        }
// set method
        void setNomorUsaha(int nomorUsaha) {
            this->nomorUsaha = nomorUsaha;
        }

        void setNamaPengusaha(string nama) {
            this->nama = nama;
        }

        void setAlamat(string alamat) {
            this->alamat = alamat;
        }

        void setNamaUsaha(string namaUsaha) {
            this->namaUsaha = namaUsaha;
        }

// get method
        int getNomorUsaha() {
            return nomorUsaha;
        }

        string getNama() {
            return nama;
        }

        string getAlamat() {
            return alamat;
        }

        string getNamaUsaha() {
            return namaUsaha;
        }

        ~Pengusaha(){};

};
